drivers/at_cmd/at_cmd.c

```diff
@@ -289,6 +291,11 @@ int at_cmd_write(const char *const cmd,
 
 void at_cmd_set_notification_handler(at_cmd_handler_t handler)
 {
+	LOG_DBG("Setting notification handler to %p", handler);
+	if ((notification_handler != NULL) && (handler != NULL)) {
+		LOG_WRN("Forgetting prior notification handler %p",
```

Comment:
Note, that with https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1163 other modules should rather use the `at_notif` dispatcher API, as the notifications can be multiplexed now.

Anyway, it could be useful to have a warning here as well.